src/World.h

```diff
@@ -817,6 +817,8 @@ class cWorld :
 		return (IsWeatherWet() && !IsBiomeNoDownfall(GetBiomeAt(a_BlockX, a_BlockZ)));
 	}
 
+	virtual bool IsWeatherWetAtBlock(Vector3i a_Pos);
```

Comment:
Add doxycomment. Document whether the Y coord must be valid, adapt code to that.